src/debugger/util.ts

```diff
@@ -28,9 +28,7 @@ function moveTokenCursorToBreakpoint(
   const coor = [...debugResponse.coor]; // Copy the array so we do not modify the one stored in state
 
   for (let i = 0; i < coor.length; i++) {
-    while (!tokenCursor.downListSkippingMeta()) {
-      tokenCursor.next();
-    }
+    tokenCursor.downListSkippingMeta();
```

Comment:
@bpringe I don't know why this wasn't just downList, so changed it to that, because I don't actually understand the while loop. And it seldom is a good idea to move non-structurally like it does inside there. OTOH, I am thinking there is a reason, so that's why tagging you here. 😄

Reply:
It's been too long for me to remember the reason that was like that 😄. However, if the existing tests pass, then we can be somewhat confident this change doesn't break anything.